config/crd/bases/nais.io_naisjobs.yaml

```
@@ -838,6 +838,15 @@ spec:
                       type: string
                   type: object
               type: object
+            restartPolicy:
+              description: RestartPolicy describes how the container should be restarted.
+                Only one of the following restart policies may be specified. If none
+                of the following policies is specified, the default one Never. Read
```

```suggestion
                of the following policies is specified, the default is used (Never). Read
```

Var litt snodig tekst, så endret litt.